index.html

```diff
@@ -0,0 +1,741 @@
+        <div class="img-details">
+          <a href="#">The 2020 Adobe Graphic Design Certification School</a>
+        </div>
+        <div class="deals-price">
+          <span class="span-1">$1497</span>
+          <span class="span-2">$49</span>
+        </div>
+      </div>
+      <div class="ads-container">
+        <img alt="lazy-load"
+             src="https://cdnp1.stackassets.com/db46f844d3bc0f538ec8bc67b1b18be252800838/store/3777e64125b8404b6b9d83e3c05ffadc23ec4911d18e7e3868d988a49714/sale_27873_primary_image.jpg">
+        <div class="img-details">
+          <a href="#">The 2020 Adobe Graphic Design Certification School</a>
+        </div>
+        <div class="deals-price">
+          <span class="span-1">$1497</span>
+          <span class="span-2">$49</span>
+        </div>
+      </div>
+      <div class="ads-container">
+        <img alt="lazy-load"
+             src="https://cdnp1.stackassets.com/db46f844d3bc0f538ec8bc67b1b18be252800838/store/3777e64125b8404b6b9d83e3c05ffadc23ec4911d18e7e3868d988a49714/sale_27873_primary_image.jpg">
+        <div class="img-details">
+          <a href="#">The 2020 Adobe Graphic Design Certification School</a>
+        </div>
+        <div class="deals-price">
+          <span class="span-1">$1497</span>
+          <span class="span-2">$49</span>
+        </div>
+      </div>
+      <div class="ads-container">
+        <img alt="lazy-load"
+             src="https://cdnp1.stackassets.com/db46f844d3bc0f538ec8bc67b1b18be252800838/store/3777e64125b8404b6b9d83e3c05ffadc23ec4911d18e7e3868d988a49714/sale_27873_primary_image.jpg">
+        <div class="img-details">
+          <a href="#">The 2020 Adobe Graphic Design Certification School</a>
+        </div>
+        <div class="deals-price">
+          <span class="span-1">$1497</span>
+          <span class="span-2">$49</span>
+        </div>
+      </div>
+    </div>
+
+
+  </section>
+</main>
+<footer>
+  <div class="social-menu">
+    <ul class="social-links">
+      <li><a href="#"><i class="fab fa-facebook-square"></i></a></li>
+      <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+      <li><a href="#"><i class="fab fa-youtube"></i></a></li>
+      <li><a href="#"><i class="fab fa-instagram"></i></a></li>
+      <li><a href="#"><i class="fab fa-pinterest-p"></i></a></li>
+      <li><a href="#"><i class="fas fa-rss"></i></a></li>
+      <li><a href="#"><i class="fas fa-envelope"></i></a></li>
+    </ul>
+    <ul class="footer-menu">
+      <li><a href="#">EVENTS</a></li>
+      <li><a href="#">ABOUT</a></li>
+      <li><a href="#">ADVERTISE</a></li>
+      <li><a href="#">JOBS</a></li>
+      <li><a href="#">CONTACT</a></li></ul>
+  </div>
+  <div class="legal">
+    <img src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg" alt="">
+    &copy; 2006–2020 The Next Web B.V. Made with &lt;3 in Amsterdam.
+    <img src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/stackpath.svg" alt="">
```

Comment:
This is just an example but there are other images that are missing a description on the alternative text attribute. 
**Actual**
```<img src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg" alt="">```
** Improved** :+1: :
```<img src="...some link..." alt="Robert De Niro">```

Having and alternate text help SEO and Screen-readers